ml-conversational-analytic-tool/githubDataExtraction.py

```diff
@@ -161,17 +155,26 @@ def getPullFeatures(self, pull):
     parser = argparse.ArgumentParser(description='Create csv for all pulls in repo')
     parser.add_argument('organization', help='Organization the repo belongs to.')
     parser.add_argument('reponame', help='Name of repo')
```

Comment:
Nothing in the help message for the `--all` option. Let's make the user know that such option exists

Reply:
I removed the --all option